rero_ils/modules/patrons/views.py

```diff
@@ -43,6 +48,50 @@
 )
 
 
+_PID_REGEX = re.compile(r'NOT\s+pid:(\w+)')
+_EMAIL_REGEX = re.compile(r'email:"(.*?)"')
```

Comment:
should be written as `_EMAIL_REGEX = re.compile(r'email:"\s*(.*?)\s*"')` to avoid problem if `email` is prefixed or suffixed with spaces